rust-lisa/src/main/java/it/unipr/cfg/expression/comparison/RustOrExpression.java

```diff
@@ -14,6 +14,8 @@
 import it.unive.lisa.program.cfg.statement.BinaryExpression;
```

Comment:
Does Rust implement a short-circut/lazy semantics? If so, the or semantics should be revised.

Reply:
An implementation was added in commit 3a08c7d. Currently we are unable to testi it.